cmd/minikube/cmd/ssh.go

```diff
@@ -49,12 +50,21 @@ var sshCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("Error getting config", err)
 		}
-		// TODO: allow choice of node to ssh into
-		cp, err := config.PrimaryControlPlane(cc)
-		if err != nil {
-			exit.WithError("Error getting primary control plane", err)
+		var n *config.Node
+		if nodeName == "" {
+			cp, err := config.PrimaryControlPlane(cc)
+			if err != nil {
+				exit.WithError("Getting primary control plane", err)
+			}
+			n = &cp
+		} else {
+			n, _, err = node.Retrieve(cc, nodeName)
+			if err != nil {
+				out.FailureT("Node {{.nodeName}} does not exist.", out.V{"nodeName": nodeName})
+				exit.WithError("", err)
```

Comment:
Combine these two calls into one which generates the proper exit code:

`exit.WithCodeT(exit.Unavailable, "node ... does not exist")`

Reply:
done